app/models/micropost.rb

```diff
@@ -0,0 +1,15 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope ->{order(created_at: :desc)}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
```
và nói cho c sự khác biêt nhé